datasets/iris.csv

```diff
@@ -149,4 +149,3 @@ sepal_length,sepal_width,petal_length,petal_width,species
 6.5,3.0,5.2,2.0,Iris-virginica
 6.2,3.4,5.4,2.3,Iris-virginica
 5.9,3.0,5.1,1.8,Iris-virginica
-
```

Comment:
empty line is causing an extra row to be added to data frame and its related tests are failing..